### PR TITLE
Release 0.9.0

### PR DESCRIPTION
**Pull request by hunner**
Summary:
This release adds fixes for rspec-puppet 2.0 and json linting for metadata.json

Features:
- Add json linting for metadata.json (adds dep on metadata-json-lint gem)
- Document using references in fixtures

Bugfixes:
- FUTURE_PARSER=yes working with rspec-puppet 2.0
- Symlinks breaking on windows
- rspec as a runtime dependency conflicting with rspec-puppet
- root stub for testing execs

### CODE REVIEW

**Comment by logicminds** [comment]
Just wasted 2 hours again chasing the windows symlink issue.  Please merge this.  @hunner

**Comment by logicminds** [comment]
Actually, this is also needed before the release #87

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,17 @@
+2014-12-05 - Release 0.9.0
```
Comment (cmurphy) [inline CHANGELOG]: Want to update this date?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,17 @@
...
+This release adds fixes for rspec-puppet 2.0 and json linting for metadata.json
+
+Features:
+- Add json linting for metadata.json (adds dep on metadata-json-lint gem)
+- Document using references in fixtures
+
+Bugfixes:
+- FUTURE_PARSER=yes working with rspec-puppet 2.0
+- Symlinks breaking on windows
+- rspec as a runtime dependency conflicting with rspec-puppet
```
Comment (logicminds) [inline CHANGELOG]: Isn't there a dependency issue between puppet blacksmith and metadata lint?

Comment (domcleal) [inline CHANGELOG]: It's been fixed, metadata lint 0.0.6 reverted the problematic code.

**Comment by logicminds** [comment]
@hunner @cmurphy Any chance we can get this merged today during the triage event.  This is holding quite a few people up now.

**Pull request merged**